scripts/zephyr_module.py

```diff
@@ -197,6 +197,8 @@ def main():
             # workspace. Such setup is allowed, as west may be installed
             # but the project is not required to use west.
             projects = []
+    else:
+        projects = args.modules
```

Comment:
```suggestion
        projects = args.modules.copy()
```
I think making a copy of list will be better, so that appending more items to `projects` does not affect `args.modules`.

Reply:
Sure, `args.modules` is not used after that, but why not. Fixed.